cmake/amr-wind-utils.cmake

```diff
@@ -79,6 +79,7 @@ macro(init_amrex_hydro)
   else()
     set(CMAKE_PREFIX_PATH ${AMReX-Hydro_DIR} ${CMAKE_PREFIX_PATH})
     find_package(AMReX-Hydro CONFIG REQUIRED)
+    add_subdirectory(${AMReX-Hydro_DIR} build)
```

Comment:
Is this needed? Feels strange

Reply:
I needed it to compile with an external AMReX-Hydro, but it may just be my inexperience with CMake. I will look into it..

Reply:
I have removed this line. Something needs to be updated probably in Hydro for AW to compile with an external Hydro.